inst/doc/Processing_Controlled_Data_With_Bioconductor.md

```diff
@@ -1,33 +1,23 @@
-# Accessing low level *controlled* data with R/Bioconductor
+# Accessing low level *controlled* data
 
-To access controlled data, we need membership to the google group called isb-cgc-cntl (compared to isb-cgc-open).
-To join the group, we need to verify our dbGaP credentials:
+To access controlled data, you need to be a member of the isb-cgc-cntl google group (compared to isb-cgc-open).
+To join the group, you need to verify your dbGaP credentials:
 
-(a) sign in to isb-cgc.appspot.com
+(a) Sign in to isb-cgc.appspot.com
```

Comment:
Consider updating the style to an ordered or unordered list using [Markdown syntax](https://help.github.com/articles/markdown-basics/).